core/evancore.vue.libs/src/components/swipe-panel/swipe-panel.scss

```diff
@@ -24,7 +24,7 @@
 
 .evan-swipe-panel {
   position: relative;
-  z-index: 1000;
+  z-index: 1002;
```

Comment:
I couldn't find any other z-index for value 1000, is there a reason why it's incremented?

Reply:
We should really introduce some z-index guide lines :D https://medium.com/science-journal/z-index-organization-in-css-5913fd4c25c9

Reply:
The new callout and its overlay use that. And yeah, z-index management is a pain :D